### DISCUSSION TITLE
Wie sollte add access bzw. manage access... übersetzt werden

### DISCUSSION THREAD

**Discussion by Langlaeufer** [Helium314/SCEE#508] [Polls]
Eine Frage an die deutschsprachigen Mitstreiter hier.
Derzeit wird add access mit "Zugriff hinzufügen" übersetzt. Mir fiel es reichlich schwer zu erkennen, was sich hinter der Funktion "Zugriff hinzufügen" verbirgt. In Anbetracht der Tatsache, das sich der Editor an Experten richtet würde ich vorschlagen Tag-Namen wie "access" überhaupt nicht zu übersetzen.
(In view of the fact that the editor is aimed at experts, I would suggest not translating Tag-Names like "access" at all.)

**Comment by mcliquid** [comment]
Meine Stimme ist abgeleitet von der deutschen Wikidefintion: "Beschreibt rechtliche Zugangsregelungen eines Elements."

**Reply by Langlaeufer** [reply]
Danke dir. Die Übersetzung halte ich auch für die beste. Nur stellt sich die Frage was besser verständlich ist für einen OSM-Experten

"Zugang hinzufügen..." oder "access hinzufügen..."

**Reply by mcliquid** [reply]
Wäre es zu lang für "Zugang (`access`)" als Kombination?

**Reply by Langlaeufer** [reply]
Würde mir auch gefallen, falls es nicht zu lang ist.

**Comment by Langlaeufer** [comment]
Mir fällt noch eine Variante ein: man könnte das Wort Tags aufnehmen um die Funktion deutlicher zu machen.
z.B.  Zugangs-Tags hinzufügen/bearbeiten/verwalten...
